src/app/models/sds-namespace.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * SDS Namespace representation, see
- * {@link https://ocs-docs.osisoft.com/Content_Portal/Documentation/Management/Account_Namespace_1.html|OCS Documentation},
+ * {@link https://ocs-docs.osisoft.com/Content_Portal/Documentation/Management/Account_Namespace_1.html|ADH Documentation},
```

Comment:
separately, I don't really like the idea of saying "ADH Documentation" when said documentation doesn't mention ADH anywhere. In a literal sense, this is still "OCS" documentation. Rebranding this before the docs are rebranded seems a bit awkward

Reply:
Turns out they just launched the ADH portion of the docs so with the new links this should be good